tools/syz-check/check.go

```diff
@@ -625,25 +625,33 @@ func checkNetlinkAttr(typ *prog.StructType, policy nlaPolicy) string {
 	return ""
 }
 
-func minTypeSize(typ prog.Type) int {
-	if !typ.Varlen() {
-		return int(typ.Size())
+func minTypeSize(t prog.Type) int {
+	if !t.Varlen() {
+		return int(t.Size())
 	}
-	if str, ok := typ.(*prog.StructType); ok {
+	switch typ := t.(type) {
+	case *prog.StructType:
```

Comment:
Don't we have to take care of the OverlayField here? It definitely affects min size.

Reply:
Done